src/test/kotlin/com/gamingbacklog/api/gamingbacklogapi/GamingBacklogApiApplicationTests.kt

```diff
@@ -1,13 +1,13 @@
-package com.gamingbacklog.api.gamingbacklogapi
-
-import org.junit.jupiter.api.Test
-import org.springframework.boot.test.context.SpringBootTest
-
-@SpringBootTest
-class GamingBacklogApiApplicationTests {
-
-	@Test
-	fun contextLoads() {
-	}
-
-}
+//package com.gamingbacklog.api.gamingbacklogapi
```

Comment:
Do we have a ticket for commenting this back in?

Reply:
nope, but fixed it :)